### PR TITLE
`Spelling and Grammar Errors` Feature

### LINKED ISSUE #83: US-03: As a user, I want the system to be able to handle any kind of text or grammatical errors, so that the summary is easy to understand.
**Tasks:**
- [x] Identify common text and grammatical errors.
- [x] Implement error detection mechanisms.
- [ ] Develop error correction methods.
- [ ] Ensure corrected text maintains clarity.
- [ ] Test the error handling system.

**Acceptance Criteria:**
- [ ] The system correctly identifies common text and grammatical errors.
- [ ] Error correction is accurately implemented without compromising text clarity.
- [ ] The error handling system passes tests with different types of input text.
- [ ] Users find the text corrections to be meaningful and easy to understand.
- [x] Frontend shows when there are spelling mistakes.
- [ ] Backend (Facebook model) not produce (output) spelling mistakes

### PR DESCRIPTION
**Pull request by gracebilliris**
This PR includes the following tasks:
- [x] Adding `spellCheck={true}` to show any errors on _textarea_ (text to be summarised).
- [x] Updating `placeholder` text.

### CODE REVIEW

File: cypress/e2e/8EmotionAnalysis.cy.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+    it("Testing Emotion Analysis works", () => {
+        cy.get("#inputted-text").type(
+            "Machine learning (ML) is an umbrella term for solving problems for which development of algorithms by human programmers would be cost-prohibitive, and instead the problems are solved by helping machines discover their own algorithms, without needing to be explicitly told what to do by any human-developed algorithms. Recently, generative artificial neural networks have been able to surpass results of many previous approaches. Machine-learning approaches have been applied to large language models, computer vision, speech recognition, email filtering, agriculture and medicine, where it is too costly to develop algorithms to perform the needed tasks. The mathematical foundations of ML are provided by mathematical optimisation (mathematical programming) methods. Data mining is a related (parallel) field of study, focusing on exploratory data analysis through unsupervised learning."
+        );
 
-    cy.get("#summarise-button-id").click();
+        cy.get("#summarise-button-id").click();
 
-    cy.wait(12000);
+        cy.wait(12000);
```
Comment (stephenb-coding) [inline cypress/e2e/8EmotionAnalysis.cy.js]: Does the emotion analysis require the summarise button to be clicked first? Could save some processing time if not

File: cypress/e2e/8EmotionAnalysis.cy.js [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
-    cy.get("#sentiment-button").click();
+        cy.get("#sentiment-button").click();
 
-    cy.wait(1000);
+        cy.wait(5000);
```
Comment (stephenb-coding) [inline cypress/e2e/8EmotionAnalysis.cy.js]: If you want to make this faster in cases where `#emotion-result` changes before 5 seconds have passed, you could remove `cy.wait(5000);` and use `timeout` to check if the span has updated in that time:

```
cy.get("#emotion-result", { timeout: 5000 }).should(($span) => {
    // Get the text content of the <span> element
    const spanText = $span.text();
    expect(spanText).to.not.equal("Not set");
});
```

**Review by stephenb-coding — APPROVED** [review]
Looks good, couple of potential performance improvements:

**Review by Kyrus2504 — COMMENTED** [review]
Looks good!

**Review by Kyrus2504 — APPROVED** [review]

**Pull request merged**